golem/task/taskmanager.py

```diff
@@ -164,9 +176,17 @@ def create_task(self, dictionary, minimal=False):
                                                    minimal)
         definition.task_id = CoreTask.create_task_id(self.keys_auth.public_key)
         definition.concent_enabled = dictionary.get('concent_enabled', False)
-        builder = builder_type(self.node, definition, self.dir_manager)
 
-        return builder.build()
+        task = builder_type(self.node, definition, self.dir_manager).build()
+        task_id = task.header.task_id
+
+        logger.info("Creating task %r (%s)", type(task), task_id)

```

Comment:
could be formatted as kwargs:
```suggestion
        logger.info("Creating task. type=%r, id=%s", type(task), task_id)
```